code/modules/surgery/surgery_step.dm

```diff
@@ -116,10 +116,10 @@
 		// Blood splatters on tools and user
 		if(tool && prob(20))
 			tool.add_mob_blood(target)
-			to_chat(user, span_warning("Your [tool] gets covered [target]'s blood "))
+			to_chat(user, span_warning("\The [tool] gets covered in [target]'s blood "))
```

Comment:
Does [tool] not automatically add "the"?

Reply:
It does